### DISCUSSION TITLE
Ability to hide linked references on a published page

### DISCUSSION THREAD

**Discussion by nikydix** [roam-garden/i#89] [Feature requests & Ideas]
Hey, I would love the ability to hide linked references on a Roam garden page, that reflect the filtered options on a Roam page. 

That way, some linked references could be filtered out 
Or if there's any other way of hiding linked references that avoid many orphaned link references would be fab! :
<img width="544" alt="Screenshot 2021-10-02 at 19 34 15" src="https://user-images.githubusercontent.com/85072306/135728105-b1d8d2cb-f8ea-4920-a8f0-8071aa6f61a4.png">
)

**Comment by p3ob7o** [comment]
There's a bit of a brutal way:

`.references-block {
  display:none;
}`

That hides them all at once.
